examples/ksonnet/components/10-configmaps.jsonnet

```diff
@@ -234,7 +233,7 @@ local e2eConfig = {
     containers: [
       {
         name: "e2e",
-        image: "gcr.io/heptio-images/kube-conformance:latest",
+        image: "gcr.io/heptio-images/kube-conformance:v1.8.0",
```

Comment:
How is this related to the PodLogs?

Reply:
I was testing latest changes, and in a similar area.  Just expedience.